src/controllers/album.js

```diff
@@ -0,0 +1,101 @@
+const getDb = require("../services/db");
+
+// creates a new album
```

Comment:
The purpose of comments is generally to explain something that the code cant explain by itself on first reading (eg if you have something particularly complex or unusual that might mislead someone trying to make sense of it). The 'heading' type comments at the top of each function in this file here are not really adding much other than a bit of noise so could be deleted without any negative impact